src/main/java/net/sf/jabref/specialfields/SpecialFieldsUtils.java

```diff
@@ -85,14 +85,15 @@ public static void updateField(SpecialField e, String value, BibEntry be, NamedC
         UpdateField.updateField(be, e.getFieldName(), value, nullFieldIfValueIsTheSame)
                 .ifPresent(fieldChange -> ce.addEdit(new UndoableFieldChange(fieldChange)));
         // we cannot use "value" here as updateField has side effects: "nullFieldIfValueIsTheSame" nulls the field if value is the same
-        SpecialFieldsUtils.exportFieldToKeywords(e, be.getField(e.getFieldName()), be, ce);
+        SpecialFieldsUtils.exportFieldToKeywords(e, be.getFieldOptional(e.getFieldName()), be, ce);
     }
 
     private static void exportFieldToKeywords(SpecialField e, BibEntry be, NamedCompound ce) {
-        SpecialFieldsUtils.exportFieldToKeywords(e, be.getField(e.getFieldName()), be, ce);
+        SpecialFieldsUtils.exportFieldToKeywords(e, be.getFieldOptional(e.getFieldName()), be, ce);
     }
 
-    private static void exportFieldToKeywords(SpecialField e, String newValue, BibEntry entry, NamedCompound ce) {
+    private static void exportFieldToKeywords(SpecialField e, Optional<String> newValue, BibEntry entry,
```

Comment:
I think the name of the parameter already states that one shouldn't use an optional here? Why should a newValue be optional?